frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -214,8 +230,8 @@ action.GitMachete.ResetCurrentBranchAction.GitMacheteToolbar.text=Reset Current
 action.GitMachete.ResetCurrentBranchAction.description=Reset current branch to its remote tracking branch
```

Comment:
Completely unrel, I can see a slight fix is needed to the reset info dialog:

![image](https://user-images.githubusercontent.com/3383210/88818656-38cae080-d1bf-11ea-9d04-3edaf862fc6b.png)

Should rather be `git reflog <branch-name>`, not `git reflog [HEAD]`.

Reply:
`s/performs git reset --keep/will now perform git reset --keep <remote-branch-name>` for clarity

Reply:
Also, at the end... now I think more natural would be to replace the final sentence with sth like:
```
Use git reset --keep `<current-commit-SHA>` to restore the pre-reset commit.
```
instead of pointing the users to the reflog.

Reply:
![image](https://user-images.githubusercontent.com/19799111/88921709-43dd4980-d26f-11ea-84aa-c9b33330a95a.png)

Reply:
@PawelLipski  WDYT?

Reply:
Ok, maybe shorten the SHA to ~15 chars or so

Reply:
Wait, there's a mistake: not `git reset --keep <local-branch>` but `git reset --keep <remote-branch>`

Reply:
![image](https://user-images.githubusercontent.com/19799111/89157232-75eef400-d56c-11ea-8542-58f077d5d9a8.png)

@PawelLipski

Reply:
Great! let's leave as above